### ISSUE TITLE
Store how many turnips were bought

### ISSUE THREAD

**Issue by stevechadbourne** [jamesmontemagno/islandtracker-feedback#1]
It would be nice if we could capture the number of turnips bought as well as the price. This will enable the calculation of the profit made and lead to time based performance charts.

**Comment by jamesmontemagno** [comment]
Yeah! I plan on adding in "Transactions" where you can add in buy/sell data.
